eng/tools/suppressions/src/suppressions.ts

```diff
@@ -11,6 +11,8 @@ export interface Suppression {
   tool: string;
   // Output only exposes "paths".  For input, if "path" is defined, it is inserted at the start of "paths".
   paths: string[];
+  rules?: string[];
+  subRules?: string[];
```

Comment:
The readbility is not good for spec PR author, “ignore-fields” (parameters/options) is better in `sdk-tspconfig-validation` rule. For other rules, it should be other names. Then different rules must use different patterns of `sub-rules`.

Reply:
The new schema of `Suppresion` apply to all existing rules, but you prefer to make the logic to extract sub rules into `sdk-tspconfig-validation` rule, which I thought is a general logic for all rules. Looks like `sub-rules` is a specific feature only for  `sdk-tspconfig-validation` rule, do you want me to add a check when `sub-rules` is added without `sdk-tspconfig-validation` rule?